sway/meson.build

```diff
@@ -1,252 +1,255 @@
+    'commands/input/pointer_accel.c',
+    'commands/input/repeat_delay.c',
+    'commands/input/repeat_rate.c',
+    'commands/input/scroll_button.c',
+    'commands/input/scroll_factor.c',
+    'commands/input/scroll_method.c',
+    'commands/input/tap.c',
+    'commands/input/tap_button_map.c',
+    'commands/input/tool_mode.c',
+    'commands/input/xkb_capslock.c',
+    'commands/input/xkb_file.c',
+    'commands/input/xkb_layout.c',
+    'commands/input/xkb_model.c',
+    'commands/input/xkb_numlock.c',
+    'commands/input/xkb_options.c',
+    'commands/input/xkb_rules.c',
+    'commands/input/xkb_switch_layout.c',
+    'commands/input/xkb_variant.c',
 
-	'commands/output/adaptive_sync.c',
-	'commands/output/background.c',
-	'commands/output/disable.c',
-	'commands/output/dpms.c',
-	'commands/output/enable.c',
-	'commands/output/max_render_time.c',
-	'commands/output/mode.c',
-	'commands/output/position.c',
-	'commands/output/power.c',
-	'commands/output/render_bit_depth.c',
-	'commands/output/scale.c',
-	'commands/output/scale_filter.c',
-	'commands/output/subpixel.c',
-	'commands/output/toggle.c',
-	'commands/output/transform.c',
-	'commands/output/unplug.c',
+    'commands/output/adaptive_sync.c',
+    'commands/output/background.c',
+    'commands/output/disable.c',
+    'commands/output/dpms.c',
+    'commands/output/enable.c',
+    'commands/output/max_render_time.c',
+    'commands/output/mode.c',
+    'commands/output/position.c',
+    'commands/output/power.c',
+    'commands/output/render_bit_depth.c',
+    'commands/output/scale.c',
+    'commands/output/scale_filter.c',
+    'commands/output/subpixel.c',
+    'commands/output/toggle.c',
+    'commands/output/transform.c',
+    'commands/output/unplug.c',
 
-	'tree/arrange.c',
-	'tree/container.c',
-	'tree/node.c',
-	'tree/root.c',
-	'tree/view.c',
-	'tree/workspace.c',
-	'tree/output.c',
+    'tree/arrange.c',
+    'tree/container.c',
+    'tree/node.c',
+    'tree/root.c',
+    'tree/view.c',
+    'tree/workspace.c',
+    'tree/output.c',
 )
 
 subdir('desktop/shaders')
 
 sway_deps = [
-	cairo,
-	drm,
-	jsonc,
-	libevdev,
-	libinput,
-	libudev,
-	math,
-	pango,
-	pcre2,
-	glesv2,
-	pixman,
-	threads,
-	wayland_server,
-	wlroots,
-	xkbcommon,
-	xcb_icccm,
-	egl,
+    cairo,
+    drm,
+    jsonc,
+    libevdev,
+    libinput,
+    libudev,
+    math,
+    pango,
+    pcre2,
+    glesv2,
+    pixman,
+    threads,
+    wayland_server,
+    wlroots,
+    xkbcommon,
+    xcb_icccm,
+    egl,
 ]
 
 if have_xwayland
-	sway_sources += 'desktop/xwayland.c'
-	sway_deps += xcb
+    sway_sources += 'desktop/xwayland.c'
+    sway_deps += xcb
 endif
 
 executable(
-	'sway',
-	sway_sources + wl_protos_src,
-	include_directories: [sway_inc],
-	dependencies: sway_deps,
-	link_with: [lib_sway_common],
-	install: true
+    'sway',
+    sway_sources + wl_protos_src,
+    include_directories: [sway_inc],
+    dependencies: sway_deps,
+    link_with: [lib_sway_common],
+    install: true
```

Comment:
Why the removal of tabs?

Reply:
Auto linting issue on my end, will be fixed prior to merging